src/TableToolbar/index.tsx

```diff
@@ -1,12 +1,11 @@
-import React from 'react';
-
+import React, { ReactElement } from 'react';
 import { StyledTableToolbar } from './styled'
 
 import {
   PencilIcon, PlusIcon, ReplyIcon, TrashIcon,
 } from '@heroicons/react/outline';
 
-export type TableToolbarProps = {
+export type TableToolbarProps<T extends Record<string, unknown>> = {
```

Comment:
We don't need a generic type with the TableToolbarProps here. It isn't being used anywhere